### ISSUE TITLE
Scroll using two finger scrolling only one direction in osx google chrome

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#126]
```
What steps will reproduce the problem?
1. Open page with demonstration of flexpaper in google chrome 8.0.552.0 for Mac 
Os X.
2. Scroll over flexpaper with two finger scroll in any direction (either up or 
down).

What is the expected output? What do you see instead?

Insteard of scrolling in given direction, scroll goes only down.

What version of the product are you using? On what operating system?


Please provide any additional information below.

```

Original issue reported on code.google.com by `moiseev....@gmail.com` on 14 Oct 2010 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
works fine when I use os x with FlexPaper, does scrolling work for you in the 
following document?

http://bit.ly/93RHGt
```

Original comment by `erik.eng...@devaldi.com` on 15 Oct 2010 at 8:03

**Comment by GoogleCodeExporter** [comment]
```
Same problem here. One time it's scrolling normal, after that scroll broken and 
even reload of page won't help. 
```

Original comment by `moiseev....@gmail.com` on 15 Oct 2010 at 8:12

**Comment by GoogleCodeExporter** [comment]
```
I think I will hold fixing this a little while since Google Chrome 8.0.552.0 is 
in dev version and not a stable release. Keeping my eyes on it though.
```

Original comment by `erik.eng...@devaldi.com` on 16 Oct 2010 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
fixed in repository. Due to be relased in FlexPaper 1.4.1
```

Original comment by `erik.eng...@devaldi.com` on 5 Dec 2010 at 11:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hi, I have this same issue with chrome on windows 8, ubuntu 12.04 in chrome 26. 
I'm scrolling with the wheel on the mouse.
There are any new fix to solve this problem??
My version of flexpaper is 2.1.0.

Thanks, I'm looking forward for any solution to this problem
```

Original comment by `etetesta...@gmail.com` on 25 Apr 2013 at 8:34

**Comment by GoogleCodeExporter** [comment]
```
This issue has been revisited again and fixed in our upcoming 2.1.5 build 
please contact us for a prerelease. 
```

Original comment by `erik.eng...@devaldi.com` on 7 Jul 2013 at 10:33
